### ISSUE TITLE
Avalonia usage? 

### ISSUE THREAD

**Issue by vector-man** [abbaye/WpfHexEditorControl#118]
Do you think it would be too difficult to get your project running natively in Avalonia? I haven't looked at the source, thoroughly.

Avalonia: https://avaloniaui.net/

If I get the time to, I'll attempt getting an Avalonia version up (if you're busy.) Was just wondering your thoughts on the idea.

**Comment by abbaye** [comment]
I never tested but I think I should rebuild the UI for that's.

It's not impossible but I don't have the time for this actually 😕

If you want try to use in Avalonia and come back :)

**Comment by michalss** [comment]
any progress on this pls?

**Comment by abbaye** [comment]
Unfortunately I haven't worked on wpf hex editor for about 1 year.  I work 60-70 hours a week.  In fact I use my free time away from my screens.

**Comment by abbaye** [comment]
Hi @michalss 

You can check with @vector-man 😎 He worked on this 

https://github.com/vector-man/WPFHexEditorControl.Avalonia/tree/avalonia

**Comment by vector-man** [comment]
Yeah. It's unfortunately unusable at the moment. If I get some time, I'll try to get something working.

**Comment by abbaye** [comment]
Hi my friend! Yes, indeed, I've completely rebuilt the control system with 100% compatibility and it's now include the insert byte anywhere (with some bugs currently being fixed). With this new architecture, I think I'll be able to easily add support for Avalionia very soon. take care !
